frontend/packages/console-app/src/components/quick-starts/QuickStartTasks.tsx

```diff
@@ -0,0 +1,99 @@
+import * as React from 'react';
+import cx from 'classnames';
+import { Alert, Radio } from '@patternfly/react-core';
+import { QuickStartTaskItem } from './utils/quick-start-typings';
+import { TaskStatus } from './utils/quick-start-status';
+import { SyncMarkdownView } from '@console/internal/components/markdown-view';
+import TaskHeader from './QuickStartTaskHeader';
+import './QuickStartTasks.scss';
+
+type QuickStartTaskProps = {
+  tasks: QuickStartTaskItem[];
+  taskStatus: TaskStatus[];
+  activeTaskIndex: number;
+  reviewCallback: (reviewState: boolean) => void;
+};
+
+const QuickStartTasks: React.FC<QuickStartTaskProps> = ({
+  tasks,
+  taskStatus,
+  activeTaskIndex,
+  reviewCallback,
+}) => (
+  <>
+    {tasks
+      .filter((_, index) => index <= activeTaskIndex)
+      .map((task, index) => {
+        const { title, description, review, recapitulation, taskHelp } = task;
+        const currTaskStatus = taskStatus[index];
+        const isActiveTask = index === activeTaskIndex;
+        return (
+          <>
+            <TaskHeader
+              key={task.title}
+              taskIndex={index + 1}
+              title={title}
+              subtitle={`${index + 1} of ${tasks.length}`}
+              taskStatus={currTaskStatus}
+              isActiveTask={isActiveTask}
+            />
+            <SyncMarkdownView
+              styles="div {overflow-y: visible !important}"
+              key={task.title}
+              content={
+                isActiveTask
+                  ? description
+                  : currTaskStatus === TaskStatus.SUCCESS
+                  ? recapitulation.success
+                  : recapitulation.failed
+              }
+            />
+            {isActiveTask && currTaskStatus !== TaskStatus.INIT && (
```

Comment:
You also need to check if there is a review available for this task.

Reply:
done